packages/integration-tests/src/tests/api/interaction/forgot-password/sad-path.test.ts

```diff
@@ -0,0 +1,122 @@
+import { ConnectorType } from '@logto/connector-kit';
+import { InteractionEvent, SignInIdentifier } from '@logto/schemas';
+
+import { suspendUser } from '#src/api/admin-user.js';
+import {
+  patchInteractionIdentifiers,
+  putInteraction,
+  putInteractionProfile,
+  sendVerificationCode,
+} from '#src/api/interaction.js';
+import { initClient } from '#src/helpers/client.js';
+import {
+  clearConnectorsByTypes,
+  setEmailConnector,
+  setSmsConnector,
+} from '#src/helpers/connector.js';
+import { expectRejects, readVerificationCode } from '#src/helpers/index.js';
+import { enableAllVerificationCodeSignInMethods } from '#src/helpers/sign-in-experience.js';
+import { generateNewUser, generateNewUserProfile } from '#src/helpers/user.js';
+import { generatePassword } from '#src/utils.js';
+
+describe('reset password flow sad path', () => {
+  beforeAll(async () => {
+    await clearConnectorsByTypes([ConnectorType.Email, ConnectorType.Sms]);
+    await setEmailConnector();
+    await setSmsConnector();
+    await enableAllVerificationCodeSignInMethods({
+      identifiers: [SignInIdentifier.Email, SignInIdentifier.Phone],
+      password: true,
+      verify: true,
+    });
+  });
+
+  afterAll(async () => {
+    await clearConnectorsByTypes([ConnectorType.Email, ConnectorType.Sms]);
+  });
+
+  afterEach(async () => {
+    await enableAllVerificationCodeSignInMethods({
+      identifiers: [SignInIdentifier.Email, SignInIdentifier.Phone],
+      password: true,
+      verify: true,
+    });
+  });
```

Comment:
remove `enableAllVerificationCodeSignInMethods` from beforeAll, and change this to beforeEach?

Reply:
BTW, not sure, but do we need to set the signUp method here?

Reply:
This is a mistake. Initially, I wanted to test the login process after resetting the password, but this step has already been covered in the happy path and sign-in-related tests.

Removed.